example/client.rb

```diff
@@ -68,7 +68,9 @@
 
 stream.on(:close) do
   log.info 'stream closed'
-  sock.close
+
+  # keep connection open to handle push messages
+  # sock.close
```

Comment:
The push example uses multiple streams. So, the socket should stay open until all pushes finish. This seemed like the simplest way to do it.